src/kuberesources.virtualfs.ts

```diff
@@ -51,7 +51,9 @@ export class KubernetesResourceVirtualFileSystemProvider implements FileSystemPr
     async loadResource(uri: Uri): Promise<string> {
         const outputFormat = workspace.getConfiguration('vs-kubernetes')['vs-kubernetes.outputFormat'];
         const value = querystring.parse(uri.query).value;
-        const sr = await this.kubectl.invokeAsyncWithProgress(`-o ${outputFormat} get ${value}`, `Loading ${value}...`);
+        const ns = querystring.parse(uri.query).ns;
```

Comment:
querystring.parse returns an object, so you should just be able to do this once and read from the object.